### ISSUE TITLE
Release v1.2

### ISSUE THREAD

**Issue by rgieseke** [openclimatedata/pymagicc#56]
- [x] update changelog
- [x] re-run generated API docs (maybe move to separate file)
- [x] ensure examples are working

**Comment by rgieseke** [comment]
Docs with pydocmd https://github.com/NiklasRosenstein/pydoc-markdown

**Comment by rgieseke** [comment]
Docs are good enough for now, but should be improved for the next release (#57)

**Comment by rgieseke** [comment]
Notebooks checked.

**Comment by rgieseke** [comment]
#55 is probably not needed for 1.2

@znicholls  If I get your feedback on #46 and a general Ok for a new release I can put this on PyPI today!

If you could give docs, readme, changelog, code a quick check at some point today that would be great!

**Comment by znicholls** [comment]
Yes remind me at lunch

**Comment by rgieseke** [comment]
Done after #58
